src/components/summary-list/summary-list.yaml

```diff
@@ -0,0 +1,447 @@
+              - href: '#'
+                text: Change
+              - href: '#'
+                text: Mail
+              - href: '#'
+                text: Upgrade
+              - href: '#'
+                text: Charge
+              - href: '#'
+                text: Point
+              - href: '#'
+                text: Coom
+              - href: '#'
+                text: Press
+              - href: '#'
+                text: Snap
+              - href: '#'
+                text: Work
+              - href: '#'
+                text: Quick
+              - href: '#'
+                text: Erase
+        - key:
+            text: Pneumonoultramicroscopicsilicovolcanoconiosis
+          value:
+            html: |
+              <p class="govuk-body">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi quis consequat diam. Duis efficitur justo at congue iaculis. Quisque scelerisque ornare justo nec congue. Duis egestas felis nibh, eu cursus metus rutrum eget. In dictum lectus diam, dapibus ullamcorper risus gravida a. Vestibulum tempor mattis sapien, at auctor tellus dignissim non. Praesent dictum felis nec diam tempor, vel lobortis leo ultricies.</p>
+              <p class="govuk-body">Suspendisse potenti. Aliquam dictum eu ipsum sed facilisis. Maecenas hendrerit est eget ultrices venenatis. Nam ex nisl, venenatis eget molestie quis, hendrerit id tellus. Morbi et posuere ex, vel interdum sapien. Mauris ac mattis turpis, interdum eleifend erat. Morbi eget efficitur lectus. Sed suscipit laoreet ipsum et iaculis. Integer ornare ipsum quis aliquet scelerisque. Proin venenatis dictum suscipit. Nunc tristique, felis quis fermentum rhoncus, tortor augue egestas ipsum, non porttitor nulla odio vitae purus. Interdum et malesuada fames ac ante ipsum primis in faucibus.</p>
+          actions:
+            items:
+              - href: '#'
+                text: Write
+              - href: '#'
+                text: Cut
+              - href: '#'
+                text: Paste
+              - href: '#'
+                text: Save
+              - href: '#'
+                text: Load
+              - href: '#'
+                text: Check
+              - href: '#'
+                text: Quick
+              - href: '#'
+                text: Rewrite
+              - href: '#'
+                text: Plug
+              - href: '#'
+                text: Play
+              - href: '#'
+                text: Burn
+              - href: '#'
+                text: Rip
+              - href: '#'
+                text: Drag and drop
+              - href: '#'
+                text: Zip
+              - href: '#'
+                text: Unzip
+              - href: '#'
+                text: Lock
+              - href: '#'
+                text: Fill
+              - href: '#'
+                text: Curl
+              - href: '#'
+                text: Find
+              - href: '#'
+                text: View
+              - href: '#'
```

Comment:
Minor – this last link doesn't have any text, so you get an empty link at the end of the actions.